app/sass/assignments/_new_assignment.scss

```diff
@@ -11,6 +11,7 @@
     @extend .columns;
 
     height: 100%;
```

Comment:
This is causing some overflow issues since the page title is above this div and this div's height is 100%. so our height is 100% + height of the page title